docs/machine-learning/resources/transforms.md

```diff
@@ -1,168 +1,103 @@
-| <xref:Microsoft.ML.Transforms.LoadTransform> | Loads specific transforms from the specified model file. Allows for 'cherry picking' transforms from a serialized chain, or to apply a pre-trained transform to a different (but still compatible) data view. |
-| <xref:Microsoft.ML.Transforms.Text.NgramExtractingTransformer> | Produces a bag of counts of ngrams (sequences of consecutive values of length 1-n) in a given vector of keys. It does so by building a dictionary of ngrams and using the id in the dictionary as the index in the bag. |
-| <xref:Microsoft.ML.Transforms.Text.NgramExtractorTransform> | Turns a collection of tokenized text (vector of ReadOnlyMemory), or vectors of keys into numerical feature vectors. The feature vectors are counts of ngrams (sequences of consecutive tokens -words or keys- of length 1-n). |
-| <xref:Microsoft.ML.Transforms.Text.NgramHashExtractingTransformer> | Turns a collection of tokenized text (vector of ReadOnlyMemory) into numerical feature vectors using hashing. |
-| <xref:Microsoft.ML.Transforms.Text.NgramHashingTransformer> | Produces a bag of counts of ngrams (sequences of consecutive words of length 1-n) in a given text. |
-| <xref:Microsoft.ML.Transforms.Categorical.OneHotEncodingTransformer> | Converts the categorical value into an indicator array by building a dictionary of categories based on the data and using the id in the dictionary as the index in the array |
-| <xref:Microsoft.ML.Transforms.Projections.PcaTransform> | Computes the projection of the feature vector onto a low-rank subspace. |
-| <xref:Microsoft.ML.Transforms.Text.SentimentAnalyzingTransformer> | Uses a pretrained sentiment model to score input strings. |
-| <xref:Microsoft.ML.Transforms.Text.StopWordsRemovingTransformer> | Removes language-specific list of stop words (most common words) by comparing individual tokens (case-insensitive comparison) to the stopwords. |
-| <xref:Microsoft.ML.Transforms.Text.WordBagBuildingTransformer> | Produces a bag of counts of ngrams (sequences of consecutive words) in a given text. It does so by building a dictionary of ngrams and using the id in the dictionary as the index in the bag. |
-| <xref:Microsoft.ML.Transforms.Text.WordHashBagProducingTransformer> | Produces a bag of counts of ngrams (sequences of consecutive words of length 1-n) in a given text. It does so by hashing each ngram and using the hash value as the index in the bag. |
-| <xref:Microsoft.ML.Transforms.Text.WordTokenizingTransformer> | Splits the text into words using the separator character(s). |
-
-## Image model featurizers
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeMeanVariance%2A> | Subtract the mean (of the training data) and divide by the variance (of the training data) |
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeLogMeanVariance%2A> | Normalize based on the logarithm of the training data |
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeLpNorm%2A> | Normalize based on lp norm |
```

Comment:
Suggestion - add external link that may explain what lp norm is.

Reply:
Yes, I want to do this. I am going to wait until I can get more information on this.